manifests/m/MindsetInnovationInc/Enowork/1.0.7/MindsetInnovationInc.Enowork.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: MindsetInnovationInc.Enowork
+PackageVersion: 1.0.7
+PackageLocale: en-US
+Publisher: Mindset Innovation Inc.
+PublisherUrl: https://enophone.com/
+PrivacyUrl: https://enophone.com/privacy-policy/
+PackageName: Enowork

```

```suggestion
PackageName: Eno
```
This must match control panel